roles/elasticsearch/tasks/main.yml

```diff
@@ -1,16 +1,199 @@
 ---
+- name: check if certs exist
+  stat: 'path=/etc/letsencrypt/live/{{ hostname }}/privkey.pem'
+  register: certfile
+
+- name: fail if certs not present
+  fail: 'msg="/etc/letsencrypt/live/{{ hostname }}/privkey.pem not found"'
+  when: not certfile.stat.exists
+
+- name: set vars
+  ansible.builtin.set_fact:
+    elastic_running: false
+    cluster_running: false
+
+- name: check if elasticsearch already running
+  ansible.builtin.uri:
+    url: 'https://{{ hostname }}:{{ http_port }}/'
+    user: '{{ admin_user }}'
+    password: '{{ admin_password }}'
+    validate_certs: false
+    return_content: true
+  register: elastic_response
+  ignore_errors: true
+
+- name: set a var if ES is already running
+  when: "elastic_response.content is defined and 'cluster_name' in elastic_response.content"
+  ansible.builtin.set_fact:
+    elastic_running: true
```

Comment:
by convention the command, like 'ansible.builtin.set_fact' should be right after name

Reply:
Done